src/main/java/at/favre/lib/crypto/HKDF.java

```diff
@@ -190,9 +235,9 @@ HkdfMacFactory getMacFactory() {
          *                            if not provided, it is set to a array of hash length of zeros.
```

Comment:
I believe the "if not provided" line is about the salt and not the IKM

Reply:
True, this is a doc bug.